src/rotation.rs

```diff
@@ -0,0 +1,59 @@
+use vector::Vector3;
+
+
+/// Euler rotation angles in 3D space.
```

Comment:
I always thought that ZYX was the "standard". Why does `cgmath` do it the other way around?